cmd/clusterctl/internal/test/fake_objects.go

```diff
@@ -51,7 +51,7 @@ type FakeCluster struct {
 	machines              []*FakeMachine
 	withCloudConfigSecret bool
 	withCredentialSecret  bool
-	topologyClass         *string
+	topology              *FakeTopology
```

Comment:
I renamed this because it's only called [topology in the api](https://github.com/kubernetes-sigs/cluster-api/blob/4350c4447ecfb7294dbe0e353f73acbf4d4da3ed/api/v1beta1/cluster_types.go#L70)

Reply:
Good catch!